src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import { Outlet } from "react-router-dom";
+import Footer from "./Footer";
+
+export default function App() {
+	return (
+		<>
+			<>
+				<Outlet />
+			</>
+			<Footer />
+		</>
+	);
+}
```

Comment:
```suggestion
export default function App() {
	return (
		<>
			<Outlet />
			<Footer />
		</>
	);
}
```
혹시 `React.Fragment`를 이렇게 사용하신 이유가 있으실까요?

Reply:
원래 저기엔 div가 들어갔었는데, header의 포지션을 만지는 과정에서 div만 삭제하고 놔뒀었나 보네요....! 수정하겠습니다!